packages/ui/components/Menu.tsx

```diff
@@ -0,0 +1,74 @@
+import classnames from 'classnames'
+import Link from 'next/link'
+
+export interface MenuItem {
+  text: string
+  url: string
+}
+
+export type MenuVariant = 'primary' | 'secondary' | 'text'
+
+export interface MenuProps {
+  items: MenuItem[]
+  variant?: MenuVariant
+  size?: 'small' | 'regular' | 'large'
+  flexJustify?: 'start' | 'end' | 'center' | 'between' | 'around' | 'evenly'
+  flexGap?: number
+  containerClasses?: string
```

Comment:
no need for these. we can just extend the html header props and pass the entire thing as otherProps.